build.gradle

```diff
@@ -31,6 +31,10 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
```

Comment:
Is the shadow plugin required for the license report?

Reply:
Apparently not, at least not with the current version of `com.github.jk1.dependency-license-report`. Don't know why this has been there before. Removed again with bd1d836.